openmls/src/lib.rs

```diff
@@ -54,11 +54,11 @@ pub mod key_packages;
 pub mod key_store;
 pub mod messages;
 pub mod schedule;
-pub mod treesync;
 
 // Private
 mod binary_tree;
 mod tree;
+mod treesync;
```

Comment:
This change again reopened issue: https://github.com/openmls/openmls/issues/722
@kkohbrok  @raphaelrobert